app/sections/view-solution-capabilities/template.njk

```diff
@@ -0,0 +1,62 @@
+{% extends 'views/includes/layout.njk' %}
+{% from 'sections/view-solution-capabilities/macro.njk' import viewSolutionCapabilities %}
+
+{% block body %}
+  <h1>View Solution Capabilities</h1>
+
+  <h3>To use the component</h3>
+  <pre><code>
+    {% verbatim %}
+      {% from 'components/view-solution-capabilities/macro.njk' import viewSolutionContactDetails %}

```

Comment:
sorry this needs to change too `viewSolutionContactDetails `
the line should be the same as line 2